src/allocator.c

```diff
@@ -31,28 +31,28 @@
 static void *
 __default_allocate(size_t size, void * state)
 {
-  (void)state;  // unused
+  RCUTILS_UNUSED(state);
```

Comment:
Sorry, since I had added the RCUTILS_UNUSED macro for use in some of my own functions I thought it would be good to just update the other place in the repository that was doing the same thing. Let me know if you want me to revert that for this pull request.

Reply:
I would highly recommend to keep the patch as small as possible to avoid any of such changes to hold back the review of the PR. Especially since we are getting close to the release.